maintenance/maintenance-template.md

```diff
@@ -5,7 +5,7 @@
 > See the [Maintenance Grants Process](https://github.com/w3f/Grants-Program#hammer_and_wrench-maintenance-grants) on how to submit a proposal.
 
 - **Team Name:** Legal name of your team (e.g. JsonCorp)
-- **Payment Address:** BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) payment address. Please also specify the currency. (e.g. 0x8920... (DAI))
+- **Payment Address:** In the case of fiat payment, please share your bank account privately with grants@web3.foundation via your contact email (see below) and enter here the date when you shared the information with us (e.g. Fiat 24.12.1971).Otherwise provide the BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) payment address. Please also specify the currency. (e.g. 0x8920... (DAI))
```

Comment:
```suggestion
- **Payment Address:** In the case of fiat payment, please share your bank account privately with grants@web3.foundation via your contact email (see below) and enter the date when you shared the information with us (e.g. Fiat 24.12.1971, 11:59) here. Otherwise, provide the BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) payment address. Please also specify the currency. (e.g. 0x8920... (DAI))
```